doc-deploy-dev/action.yml

```diff
@@ -208,36 +189,38 @@ runs:
       with:
         level: "INFO"
         message: >
-          Generate an 'index.html' for redirection to the latest stable version
-          of the documentation. If no stable version has been released, a
-          redirection to the development documentation is generated.
+          Create an index.html page copied from index.html in the stable version.
+          Update all the href and source links to point to the stable or dev version.
 
+    - name: "Check if the 'index.html' file exists"
```

Comment:
We could proceed this way:

1. Does stable/index.html exist? If so, copy this one.
2. Elif... Does dev/index.html exist? If so, copy this one.
3. Else: exit with 1 and notify user

Reply:
yes, i will combine the both steps in one https://github.com/ansys/actions/pull/413/commits/7d62a4ac2a1492a9e0b7b6720c18d996e54d6312